### PR TITLE
feature/requirement-upgrades

### PR DESCRIPTION
**Pull request by builtbywill**
* update all requirements, update sympy to 1.10
* update test cases to handle sympy changes, e.g. `\frac{1}{x}` gives `Mul(1, Pow(x, -1))` instead of just `Pow(x, -1)`, and `x-1` gives `Add(x, Mul(-1, 1))`
* update "relation list" grammar to support all group delimiters, add corresponding tests

### CODE REVIEW

**Comment by lily514** [comment]
@builtbywill is this one ready for review?

**Comment by builtbywill** [comment]
> @builtbywill is this one ready for review?

@lily514 yep!

File: tests/grouping_test.py [hunk 1]
```diff
@@ -15,13 +15,22 @@
...
 
-modifiable_delimiter_pairs = {
+example_relation_lists = [
+    ('x=1', 'y=2', [_Eq(x, 1), _Eq(y, 2)])
+]
+
+func_modifiable_delimiter_pairs = {
     '(': ')',
-    '\\lgroup': '\\rgroup',
+    '\\lgroup ': '\\rgroup ',
```
Comment (lily514) [inline tests/grouping_test.py:24]: spaces intentional here?

File: setup.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
     packages=find_packages(exclude=('tests')),
     py_modules=['asciimath_printer', 'latex2sympy'],
     install_requires=[
-        'sympy==1.4',
+        'sympy>=1.4',
```
Comment (lily514) [inline setup.py:9]: do we want this to be specific still? 1.10.1 is in the requirements files?

File: tests/grouping_test.py (hunk 1 above)
Comment (builtbywill) [reply tests/grouping_test.py:24]: @lily514 yeah most latex commands get spaces after them, was just making this match

File: setup.py (hunk 2 above)
Comment (builtbywill) [reply setup.py:9]: @lily514 yeah I guess we could, just to make sure requirements are installed correctly

**Comment by lily514** [comment]
@builtbywill I'm getting an error when I run scripts/setup.sh when it tries to compile? 
I can compile the master and develop branches fine.

image

**Comment by builtbywill** [comment]
> @builtbywill I'm getting an error when I run scripts/setup.sh when it tries to compile? I can compile the master and develop branches fine.
> 
> image

@lily514 hmm, what version of java/jdk do you have installed? I think I might have used homebrew to install `jenv` at some point to get set up

**Comment by builtbywill** [comment]
@lily514 it looks like "version 55" is equal to jdk 11. i think I have jdk 12 installed currently

**Comment by lily514** [comment]
> 

I think I got java sorted and it compiles correctly now, thanks. Looks good to me!

**Review by lily514 — APPROVED** [review]

**Pull request merged**